cms/djangoapps/contentstore/config/waffle.py

```diff
@@ -4,7 +4,7 @@
 """
 
 
-from openedx.core.djangoapps.waffle_utils import CourseWaffleFlag, WaffleFlagNamespace, WaffleSwitchNamespace
+from openedx.core.djangoapps.waffle_utils import WaffleFlag, CourseWaffleFlag, WaffleFlagNamespace, WaffleSwitchNamespace
```

Comment:
(pylint says line too long)

Reply:
Whoops.  I had accidentally turned off my linters!  Fixed.  (And rebased, just in case.)